packages/material/scss/_variables.scss

```diff
@@ -521,6 +521,49 @@ $card-actions-padding-y: ($card-padding-y - $button-padding-y) !default;
 $card-actions-border-width: 0 !default;
 
 
+// Chat
+$chat-padding-x: 16px !default;
+$chat-padding-y: 16px !default;
+$chat-spacing-x: 8px !default;
+$chat-spacing-y: 16px !default;
```

Comment:
Not sure what the difference between the `-padding` and `-spacing` is. From the code, it appears that the `-spacing` is used between the various chat elements? If so, using the name `-item-spacing` might be a better fit.